app/screens/select_team/add_team_item.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {Text, View} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import TouchableWithFeedback from '@components/touchable_with_feedback';
+import {useTheme} from '@context/theme';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            height: 64,
+            marginBottom: 2,
+        },
+        touchable: {
+            display: 'flex',
+            flexDirection: 'row',
+            borderRadius: 4,
+            alignItems: 'center',
+            height: '100%',
+            width: '100%',
+        },
+        text: {
+            color: theme.sidebarText,
+            marginLeft: 16,
+            ...typography('Body', 200),
+        },
+        icon_container_container: {
+            width: 40,
+            height: 40,
+        },
+        icon_container: {
+            width: '100%',
+            height: '100%',
+            alignItems: 'center',
+            justifyContent: 'center',
+            backgroundColor: changeOpacity(theme.sidebarText, 0.16),
+            borderRadius: 10,
+        },
+        icon: {
+            color: theme.sidebarText,
+            fontSize: 24,
+        },
+    };
+});
+
+export default function AddTeamItem() {
+    const theme = useTheme();
+    const styles = getStyleSheet(theme);
+    const intl = useIntl();
+    const onPress = useCallback(async () => {
+        // TODO https://mattermost.atlassian.net/browse/MM-43622
+        //goToScreen(Screens.CREATE_TEAM, 'Create team');
```

Comment:
Perhaps uncomment this line so that there is an alert being displayed when users tap on the button?